idaes/core/base/tests/test_control_volume_1d.py

```diff
@@ -3682,3 +3682,129 @@ def test_report():
 
     with pytest.raises(NotImplementedError):
         m.fs.cv.report()
+
+
+@pytest.mark.unit
+def test_estimate_states_forwards():
+    m = ConcreteModel()
+    m.fs = Flowsheet(dynamic=False)
+    m.fs.pp = PhysicalParameterTestBlock()
+
+    m.fs.cv = ControlVolume1DBlock(
+        property_package=m.fs.pp,
+        transformation_method="dae.finite_difference",
+        transformation_scheme="BACKWARD",
+        finite_elements=10,
+    )
+
+    m.fs.cv.add_geometry(length_domain_set=[0, 0.2, 0.5, 0.7, 1.0])
+    m.fs.cv.add_state_blocks(has_phase_equilibrium=False)
```

Comment:
Do we want a `length_domain_set` inconsistent with the number of specified finite elements? Also, don't we want to apply the discretizing transformation before testing?

Reply:
Not for this case - this is just to add some points so we get multiple StateBlocks when we construct the model to make sure the values propagate properly. The test only calls the `estimate_state` method, so we don;t actually solve anything.